app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -46,6 +46,11 @@ def start
     image = target_entity
     return queue_signal(:abort_job, "no image found", "error") unless image
     return queue_signal(:abort_job, "cannot analyze non docker images", "error") unless image.docker_id
+    return queue_signal(
+      :abort_job,
+      "smartproxy role missing",
+      "error"
+    ) unless MiqServer.my_server.has_active_role?("smartproxy")
```

Comment:
@moolitayer I don't understand why it needs both roles.  I don't think we support routing to a server with both roles unless you put one one role on the initial queue item and when a server picks it up, requeue it if the other role is not active.